src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -330,22 +330,28 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
                 ? new OrderedDictionary()
                 : new OrderedDictionary(StringComparer.OrdinalIgnoreCase);
 
-            foreach (ScriptFile file in referencedFiles)
+            foreach (ScriptFile scriptFile in referencedFiles)
             {
-                fileMap.Add(file.FilePath, file);
+                if (!fileMap.Contains(scriptFile.FilePath))
```

Comment:
You could also use the indexer:
```
fileMap[scriptFile.FilePath] = scriptFile;
```
That creates the entry if it doesn't exist and writes a new value if it does exist.